genshin/utility/geetest.py

```diff
@@ -20,6 +22,15 @@
 -----END PUBLIC KEY-----
 """
 
+CN_LOGIN_KEY_CERT = b"""
+-----BEGIN PUBLIC KEY-----
+MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDDvekdPMHN3AYhm/vktJT+YJr7
+cI5DcsNKqdsx5DZX0gDuWFuIjzdwButrIYPNmRJ1G8ybDIF7oDW2eEpm5sMbL9zs
+9ExXCdvqrn51qELbqj0XxtMTIpaCHFSI50PfPpTFV9Xt/hmyVwokoOXFlAEgCn+Q
+CgGs52bFoYMtyi+xEQIDAQAB
+-----END PUBLIC KEY-----
+"""
+
```

Comment:
It is not CN-specific actually. This key is used in old login endpoint for OS too so it would be better to name it other way but since in genshin.py it is currently used for CN endpoint only... Guess it's fine.